### PR TITLE
use new-repo tool to try to start openshift-release stuff

### PR DESCRIPTION
**Pull request by deads2k**
I still have to request my repo separately, right?

/assign @stevekuznetsov

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Yes, request the repo separately. The content here (obviously) looks ok

File: ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+base_images:
+  base:
+    name: "4.3"
```
Comment (tnozicka) [inline ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml]: master is 4.4

File: ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    namespace: ocp
+    tag: base
+binary_build_commands: make
```
Comment (tnozicka) [inline ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml]: make build --warn-undefined-variables

File: ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.12
```
Comment (tnozicka) [inline ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml]: might want to use 1.13 singe we are bumping master to that version

File: ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.4"
+  namespace: ocp
+tests:
```
Comment (tnozicka) [inline ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml]: test_binary_build_commands: make build GOFLAGS='-race'

File: ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+- as: e2e-aws
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
+  openshift_installer:
+    cluster_profile: aws
+- as: unit
```
Comment (tnozicka) [inline ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml]:
```
- artifact_dir: /tmp/artifacts
  as: unit
  commands: |
    mkdir -p /tmp/artifacts
    make test-unit JUNITFILE=/tmp/artifacts/junit_report.xml
  container:
    from: test-bin
```

File: ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.4"
+  namespace: ocp
+tests:
+- as: e2e-aws
```
Comment (tnozicka) [inline ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml]: I think you are required to have also the upgrade job

**Comment by deads2k** [comment]
> missing verify target and images
> 
> @deads2k I think you'd have easier time taking one of our existing operators config and just using sed

/assign @stevekuznetsov 

looks like @tnozicka sees some issues.

**Comment by tnozicka** [comment]
on @deads2k request create a PR for setting up the repo in https://github.com/openshift/release/pull/6301

**Pull request closed without merging**